eleventy.config.js

```diff
@@ -12,43 +12,24 @@ https://github.com/fluid-project/handbook.floeproject.org/raw/main/LICENSE.md.
 
 "use strict";
 
-var slug = require("github-slugger").slug;
-
 const fluidPlugin = require("eleventy-plugin-fluid");
-const fs = require("fs");
-const { EleventyRenderPlugin } = require("@11ty/eleventy");
-const MarkdownIt = require("markdown-it");
 const navigationPlugin = require("@11ty/eleventy-navigation");
 const syntaxHighlightPlugin = require("@11ty/eleventy-plugin-syntaxhighlight");
 
 // Import transforms
-const htmlMinTransform = require("./src/transforms/html-min-transform.js");
-const parseTransform = require("./src/transforms/parse-transform.js");
+const parseTransform = require("./src/_transforms/parse-transform.js");
 
 // Import data files
 const siteConfig = require("./src/_data/config.json");
-const getResourceLinks = require("./src/utils/getResourceLinks.js");
-const getArticleContents = require("./src/utils/getArticleContents.js");
-const getContentsFromNavKey = require("./src/utils/getContentsFromNavKey.js");
+const getResourceLinks = require("./src/_utils/getResourceLinks.js");
+const getArticleContents = require("./src/_utils/getArticleContents.js");
+const getContentsFromNavKey = require("./src/_utils/getContentsFromNavKey.js");
+const slugify = require("@sindresorhus/slugify");
 
 module.exports = function (config) {
```

Comment:
Let's update this to:

```suggestion
module.exports = function (eleventyConfig) {
```

And then update the `config` variable to `eleventyConfig` throughout.